app/views/email_alert_subscriptions/new.html.erb

```diff
@@ -37,10 +37,18 @@
       <% end %>
 
       <% if @signup_presenter.body %>
-        <%= render 'govuk_publishing_components/components/govspeak', content: @signup_presenter.body.html_safe %>
+        <div class="govspeak-wrapper">
+          <%= render 'govuk_publishing_components/components/govspeak', content: @signup_presenter.body.html_safe %>
+        </div>
       <% end %>
 
-      <%= submit_tag 'Create subscription', class: "button" %>
+      <%= render "govuk_publishing_components/components/button", {
+        text: "Create subscription",
+        margin_bottom: true,
+        data_attributes: {
+          "disable-with": "Create subscription"
```

Comment:
I'm not sure this is going to make any difference because we don't include the javascript bits that hook into this.

When it does work, the button text will change to the `disable-with` value, and the button will be disabled to prevent double clicks.  I'm not sure we're too worried about that here anyway, because we're not actually signing people up to anything at this point.

A reasonable walkthrough of the rails-ujs stuff is here: https://m.patrikonrails.com/a-definitive-guide-to-railss-unobtrusive-javascript-adapter-ef13bd047fff

Reply:
If we did want to disable it the govuk-frontend does include JavaScript to do that https://github.com/alphagov/govuk-frontend/releases/tag/v2.8.0  and more specifically https://github.com/alphagov/govuk-frontend/pull/1018.

Reply:
Great, thanks both, have removed this. Was being paranoid.